packages/teleterm/src/mainProcess/protocolHandler.ts

```diff
@@ -0,0 +1,55 @@
+import { protocol } from 'electron';
+import { fileURLToPath } from 'node:url';
+import * as path from 'path';
+import fs from 'fs';
+import Logger from 'teleterm/logger';
+
+const disabledSchemes = [
+  'about',
+  'content',
+  'chrome',
+  'cid',
+  'data',
+  'filesystem',
+  'ftp',
+  'gopher',
+  'javascript',
+  'mailto',
+];
+
+const logger = new Logger('WebHandlerProtection');
+
+// intercept, clean, and validate the requested file path.
+export function enableWebHandlerProtection({
+  installPath,
```

Comment:
I've just realized that we don't need that argument too, we can use `app.getAppPath()` directly.